src/mono/mono/metadata/mono-private-unstable.h

```diff
@@ -14,6 +14,12 @@
 
 #include <mono/utils/mono-publib.h>
 
+typedef MonoGCHandle MonoAssemblyLoadContextGCHandle;
 
+MONO_API MONO_RT_EXTERNAL_ONLY
+MonoAssembly *mono_assembly_load_full_alc (MonoAssemblyName *aname, const char *basedir, MonoImageOpenStatus *status, MonoAssemblyLoadContextGCHandle alc_gchandle);
+
```

Comment:
Perhaps drop the _full_ part ? And put the ALC as the first argument since this is kinda like a method call.